social_security.py

```diff
@@ -51,6 +54,17 @@
     2017 :      0.0
 }
 
+try:
+    namespaces = {'osss': 'http://ssa.gov/osss/schemas/1.0'}
```

Comment:
I like your idea to be able to parse out the values out of XML.
Perhaps an improvement would be to check if a command line parameter has been passed to the script, and use that as the filename to read from?

Reply:
I purposely left that out as to not add complexity to such a simple program. The intent was to use a default convention that would work if present. While allowing for the script to still function as you had it with hand populated values if desired.